src/main/java/Appointment.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Appointment {
+    static int id = 0;
+
+    static DateTimeFormatter dataTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+    public final int appointmentId;
+    public String petName;
+
+    private LocalDateTime appointmentDate;
+    private AppointmentStatus appointmentStatus;
+    // private Service service;
+
+    // Service class not implemented yet
+    // How to create a LocalDateTime:
+    // LocalDateTime newDate = LocalDateTime.of(2021, Month.NOVEMBER, 19, 1, 1, 45);
+    public Appointment(String petName, LocalDateTime appointmentDate, Service service) {
+        this.appointmentId = ++id;
```

Comment:
wait just wondering why to use two ids here hhh, maybe another to be renamed as count? or just do not need id at all cuz there's a dynamic array?